core/gather/driver/target-manager.js

```diff
@@ -156,17 +156,19 @@ class TargetManager extends ProtocolEventEmitter {
   /**
    * Returns a listener for all protocol events from session, and augments the
    * event with the sessionId.
+   * @param {string} targetType
```

Comment:
```suggestion
   * @param {LH.Protocol.TargetType} targetType
```

this can be used to check that the type is being narrowed to just the types we care about since it's checked before `_getProtocolEventListener` is called (code below doesn't complain about `string` -> `'page'|'iframe'|'worker'` because of the cast to `RawEventMessage`)